### PR TITLE
GDC updates

### PR DESCRIPTION
**Pull request by tomkralidis**
Initial GDC updates (not sure that the alert definitions are correct, however).

cc @golfvert

### CODE REVIEW

**Review by kaiwirt — APPROVED** [review]
Is 1 minute until a critical alert is raised when the connection to one center drops ok? In the cache i have one hour for critical alerts.

File: alert-rules/gdc.yml [hunk 1]
```diff
@@ -1,2 +1,9 @@
 groups:
-  - name: GlobalDiscoveryCatalog
+  - name: GlobalDiscoveryCatalogue
+    rules:
+      # Connection to a centre is lost
```
Comment (golfvert) [inline alert-rules/gdc.yml:4]: # Connection to a Global Broker is lost

File: alert-rules/gdc.yml [hunk 2]
```diff
@@ -1,2 +1,9 @@
...
+      - alert: Connection_lost_from_centre
+        expr: group by (centre_id,report_by) (wmo_wis2_gdc_connected_flag == 0) > 0
```
Comment (golfvert) [inline alert-rules/gdc.yml:6]: expr: group by (centre_id) (wmo_wis2_gdc_connected_flag{"centre-id" =~ ".*global-broker"} == 0) > 0

File: alert-rules/gdc.yml [hunk 3]
```diff
@@ -1,2 +1,9 @@
...
+        for: 1m
```
Comment (golfvert) [inline alert-rules/gdc.yml:7]: for: 10m

**Review by golfvert — CHANGES_REQUESTED** [review]
Proposed changes

**Comment by golfvert** [comment]
I agree with Kai. 1m is too low.
Considering the role of GDC, they subscribe to GB for messages and download metadata from GC, we could be more specific.
I am suggesting something for GB.
We need, I think, the equivalent for GC.
So, adding a metric:

`
wmo_wis2_gdc_downloaded_errors_total | centre_id\|report_by | Number of download errors for metadata | -
`

And adding 

`
      # More than 1 download errors from Global Cache in the last hour
      - alert: Download_errors_from_global_cache
        expr: group by (centre_id) (delta (wmo_wis2_gdc_downloaded_errors_total[1h]{"centre-id" =~ ".*global-cache"}) > 0) > 0
        for: 1h
        labels:
          severity: warning
`

Syntax to be checked :)

**Review by kaiwirt — APPROVED** [review]

**Pull request merged**